packages/babel-plugin-transform-parameters/test/fixtures/parameters/iife-this-9385/output.js

```diff
@@ -22,10 +22,8 @@ function () {
     key: "invite",
     value: function invite() {
       var options = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
-      return function () {
-        var privacy = options.privacy || "Private";
-        console.log(this);
-      }.apply(this);
+      var privacy = options.privacy || "Private";
+      console.log(this);
```

Comment:
Why did this change? It seems unrelated to the change.

Reply:
There's an explanation here: https://github.com/babel/babel/pull/9601/files#r260984879

Reply:
ah, I see 👍